src/redux/books/books.js

```diff
@@ -0,0 +1,23 @@
+// Actions
+const ADD_BOOK = 'bookstore/booksReducer/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/booksReducer/REMOVE_BOOK';
+
+// Reducer
+export default function booksReducer(state = [], action = {}) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+    case REMOVE_BOOK:
+      return [...state.filter((book) => (book.id !== action.payload))];
+    default: return state;
+  }
+}
+
+// Action Creators
+export function addBook(book) {
+  return { type: ADD_BOOK, payload: book };
+}
+
+export function removeBook(bookID) {
+  return { type: REMOVE_BOOK, payload: bookID };
+}
```

Comment:
- [ ] Kindly make sure that a new book can be  added or  removed, currently I can't remove or see book I added to the array 👍🏿

Reply:
Hi there,

This PR is now under second opinion review. Please have a bit of patience :)